src/worker/core/season/awards.ts

```diff
@@ -303,15 +304,20 @@ const saveAwardsByPlayer = async (
 			conditions,
 		);
 	}
-
 	const pids = Array.from(new Set(awardsByPlayer.map(award => award.pid)));
 	for (const pid of pids) {
-		const p = await idb.cache.players.get(pid);
+		let p = await idb.cache.players.get(pid);
+		if (!p) {
+			const p2 = await idb.getCopy.players({
+				pid: pid,
+			});
+			p = p2 as PlayerFiltered;
+		}
 		if (p) {
 			for (const awardByPlayer of awardsByPlayer) {
 				if (awardByPlayer.pid === pid) {
 					p.awards.push({
-						season: g.get("season"),
+						season: season ? season : g.get("season"),
```

Comment:
Fun new syntax! This can be `season: season ?? g.get("season")`. Or better yet, it can just be `season,` if you change line 265 to:

            season: number = g.get("season"),